src/shape.rs

```diff
@@ -211,8 +211,10 @@ impl<'a, 'b, B: Brush> partition::Selector for FontSelector<'a, 'b, B> {
     type SelectedFont = SelectedFont;
 
     fn select_font(&mut self, cluster: &mut CharCluster) -> Option<Self::SelectedFont> {
+        use fontique::QueryFamily;
```

Comment:
This should be at the top of the file with the other imports

Reply:
Done